src/main/java/io/temporal/internal/testservice/StateMachines.java

```diff
@@ -984,7 +987,7 @@ private static void scheduleActivityTask(
       ScheduleActivityTaskCommandAttributes d,
       long workflowTaskCompletedEventId) {
     RetryPolicy retryPolicy = ensureDefaultFieldsForActivityRetryPolicy(d.getRetryPolicy());
-    long expirationInterval = TimeUnit.SECONDS.toMillis(d.getScheduleToCloseTimeoutSeconds());
+    long expirationInterval = Durations.toMillis(d.getScheduleToCloseTimeout());
```

Comment:
change to use java.time.Duration and java timestamp equivalent?

Reply:
Duration